src/zenml/zen_server/rbac/endpoint_utils.py

```diff
@@ -34,13 +34,15 @@ def verify_permissions_and_create_entity(
     request_model: AnyRequest,
     resource_type: ResourceType,
     create_method: Callable[[AnyRequest], AnyResponse],
+    is_admin: bool = True,
 ) -> AnyResponse:
     """Verify permissions and create the entity if authorized.
 
     Args:
         request_model: The entity request model.
         resource_type: The resource type of the entity to create.
         create_method: The method to create the entity.
+        is_admin: If no RBAC is in place, the user must be an admin.
```

Comment:
I think the naming of this argument and the explanation in the docstring is a little confusing.

In most usages of this function, this is just set to the default value even though the user is not an admin in most cases. I don't really have a good suggestion for the argument name unfortunately (maybe `admin_status: Optional[bool] = None` and `if admin_status is False`), but at least we can explain it a little more clearly in the docstring?